ambry-api/src/main/java/com.github.ambry/config/StoreConfig.java

```diff
@@ -219,6 +236,10 @@ public StoreConfig(VerifiableProperties verifiableProperties) {
     storeStatsIndexEntriesPerSecond =
         verifiableProperties.getIntInRange("store.stats.index.entries.per.second", 240000, 1, Integer.MAX_VALUE);
     storeIndexPersistedEntryMinBytes = verifiableProperties.getInt("store.index.persisted.entry.min.bytes", 115);
+    storeReadOnlyEnableSizeThresholdPercentage = verifiableProperties.getInt(
+        storeReadOnlyEnableSizeThresholdPercentageName, 95);
+    storeReadWriteEnableSizeThresholdPercentageDelta = verifiableProperties.getInt(
```

Comment:
use `getIntInRange` and limit the possible values from based on the previous value.